### PR TITLE
Introduce S209 Circle with two origins

### PR DESCRIPTION
**Pull request by GeoffreySangston**
Part of https://github.com/pi-base/data/issues/675

This space will be a good counterexample for distinguishing injectively path connected from arc connected, when that gets changed + added. 

Just going to leave it to the currently added traits for this PR (unless any others are requested, also requesting the removal of any traits for now is fine too). The main thing for this PR is getting the right description mostly in order.

### CODE REVIEW

**Comment by prabau** [comment]
Yeah, one of these days, hopefully not too far off, I should get going on injectively path connected.

**Comment by GeoffreySangston** [comment]
> Yeah, one of these days, hopefully not too far off, I should get going on injectively path connected.

I'm energized to help, especially since I used arc connected incorrectly in [this recent MSE post](https://math.stackexchange.com/q/5009984/444923) and have been anxious about whether or not it's a good idea to bump it to the front page to correct it.

**Comment by prabau** [comment]
Ok, I'll make a plan for deployment in the next few days, will add it to the relevant issue.

File: spaces/S000209/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+uid: S000209
+name: Circle with two origins
+---
+
+Choose a point $0 \in S^1$ to be called the origin, and replace $0$ with two origins $0_1$ and $0_2$. Basic open neighborhoods of a point $x \neq 0$ are Euclidean open neighborhoods of $x$ not containing $0$. Basic open neighborhoods of each origin $0_i$ are of the form $(U\setminus\{0\})\cup\{0_i\}$ with $U$ a Euclidean open neighborhood of $0$.
+
+Let $\{1, 2\}$ have the discrete topology. $X$ is homeomorphic to the quotient space of $S^1 \times \{1, 2\}$ obtained by identifying $\langle \theta, 1 \rangle$ and $\langle \theta, 2 \rangle$ exactly when $\theta \not\equiv 0 \mod 2\pi$.
+
+$X$ is homeomorphic to the one-point compactification of {S83}.
```
Comment (Moniker1998) [inline spaces/S000209/README.md:10]: I don't like the name "one-point compactification" when used when referring to non-Hausdorff compactifications, since then we are lacking uniqueness. 
Instead, how about call it Alexandroff extension?

File: spaces/S000209/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+uid: S000209
+name: Circle with two origins
+---
+
+Choose a point $0 \in S^1$ to be called the origin, and replace $0$ with two origins $0_1$ and $0_2$. Basic open neighborhoods of a point $x \neq 0$ are Euclidean open neighborhoods of $x$ not containing $0$. Basic open neighborhoods of each origin $0_i$ are of the form $(U\setminus\{0\})\cup\{0_i\}$ with $U$ a Euclidean open neighborhood of $0$.
+
+Let $\{1, 2\}$ have the discrete topology. $X$ is homeomorphic to the quotient space of $S^1 \times \{1, 2\}$ obtained by identifying $\langle \theta, 1 \rangle$ and $\langle \theta, 2 \rangle$ exactly when $\theta \not\equiv 0 \mod 2\pi$.
```
Comment (Moniker1998) [inline spaces/S000209/README.md:8]: Not sure if this is a problem with my screen resolution, but $\not\equiv$, when not zoomed in, looks like $/$ followed by $\equiv$. Perhaps one could say, write `{\not\equiv}` to fix it.

File: spaces/S000209/properties/P000016.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+space: S000209
+property: P000016
+value: true
+---
+
+$X$ is homeomorphic to the one-point compactification of {S83}.
```
Comment (Moniker1998) [inline spaces/S000209/properties/P000016.md:7]: Same as previously, one shouldn't use the name "one-point compactification" and especially not with "the" since its not in any way the only one-point (non-Hausdorff) compactification.

File: spaces/S000209/README.md (hunk 2 above)
Comment (GeoffreySangston) [reply spaces/S000209/README.md]: How is it now?

**Review by Moniker1998 — APPROVED** [review]

**Comment by GeoffreySangston** [comment]
@david20000813 No worries, but I was leaving it open to see if @prabau requested any changes to the description, before I move onto the circle with countably infinitely many origins.

**Comment by prabau** [comment]
@GeoffreySangston I'll take a look anyway a little later.

**Comment by david20000813** [comment]
@GeoffreySangston Ah, sorry. I was just browsing through the PR list and saw that this one was approved two days ago and still hasn’t been merged yet. I wasn’t aware of the discussion in the issue and there wasn’t any discussion here, so I thought it was simply that you haven’t had the time to see this being approved and perform the merge due to the holidays. My apologies for this.

**Comment by prabau** [comment]
@GeoffreySangston I took a look.

The first paragraph is a good description.

The second paragraph was slightly confusing maybe, just after having read the first paragraph.  Since the origin point $0$ of $S^1$ was already introduced, the sudden switch to "mod 2 pi" was a little odd.  How about just saying at the end of the sentence:
"... when $\theta$ is not the origin point $0$ " ?

For the third paragraph, after having discussed "open extension", "closed extension", and other types of "extension" in other discussion, I was at first wondering what kind of other extension the Alexandroff one was.  Then I suddenly remembered it was another name for one-point compactification.
How about just using
`... homeomorphic to the [one-point compactification](https://en.wikipedia.org/wiki/Alexandroff_extension) of {S83}`

Background:
https://en.wikipedia.org/wiki/Alexandroff_extension seems to make a bid deal of using "Alexandroff extension" for the general case, and "one-point compactification" for when the starting space is a locally compact Hausdorff space (so that the resulting compactification is compact Hausdorff).
But that always sounded kind of artificial to me.  "Alexandroff extension" is terminology from Willard's problem 19A, and from there copied into wikipedia by whoever thought it was good to copy from Willard.  "Alexandrov extension" is not used in Engelking or Encycl. of Gen Top., etc.
And the Alexandrov extension is a compactification in the technical sense of the term, and it does so via adding a single point.

Thoughts?

**Comment by Moniker1998** [comment]
I disagree for the reasons I've already stated. 

For many people "compactification" includes Hausdorff in its definition. The "the" in the name "the one-point compactification" refers to fact that there is only one compactification which extends a space by one point, only one Hausdorff compactification that is. We could settle for calling it "a one-point compactification" but then this is not precise for what exactly the relation between those spaces is, and as mentioned before, many people restrict the name "compactification" to mean "Hausdorff compactification".

**Comment by GeoffreySangston** [comment]
> @GeoffreySangston I took a look.
> 
> The first paragraph is a good description.
> 
> The second paragraph was slightly confusing maybe, just after having read the first paragraph. Since the origin point 0 of S 1 was already introduced, the sudden switch to "mod 2 pi" was a little odd. How about just saying at the end of the sentence: "... when θ is not the origin point 0 " ?

One constraint is the desire to follow the structure of the description of [Line with two origins](https://topology.pi-base.org/spaces/S83). The issue is I'm conflating an equivalence class with a representative. To do it properly I think I would have to add text to the first paragraph, but I tried to do that just now it seemed strictly worse, so I've deleted that attempt and add the current small change. 

> Thoughts?

I did not know Alexandroff extension is from Willard. (Solomon Leader actually uses it in a [paper](https://link.springer.com/article/10.1007/BF01362349) predating Willard for a construction which sounds similar, but applies to a different kind of object called a 'local proximity space'. That paper does refer to 'one-point
compactification of a locally compact space' without specifying Hausdorff.)

I was always under the impression that "one-point compactification" was just a name for the famous construction. This also appears to be frequently called "Alexandroff one-point compactification". E.g., Ronald Brown's book on Groupoids has an exercise about "Alexandroff 1-point compactifications" without apparently restricting to Hausdorff spaces.

I don't doubt the claim that it's standard to restrict 'compactification' to include Hausdorff. I'm agnostic as to whether pi-base does or not, since it's simply a matter of adding in the adjective 'Hausdorff' to compactification. Some evidence for either view appears in Encyclopedia of General topology, which has articles with either convention:
-  There appears under *Wallman–Shanin Compactification*, "A compactification of a topological space X is a compact space Y that contains X as a dense subspace...  It should be noted that in this article the space Y is not required to be Hausdorff." And "Hausdorff compactification" does appear in a few other articles, suggesting they allow 'compactifications' which are not Hausdorff. The article *Realcompactness* seems to allow the more general notion.
- *Extensions of Topological Spaces* restricts to completely regular Hausdorff spaces. As does the article *The Cech–Stone Compactification*.

**Comment by prabau** [comment]
@Moniker1998 You make a good point.  There may indeed be many one-point compactifications of a space, that is compactifications obtained by adding one point to the original space.  So it's not a good idea to talk about "**the** one-point compactification".

Better use "Alexandroff one-point compactification", which does not have this ambiguity, and which also seems to be widely used.

As for the term "compactification" in general, Engelking uses compact to mean compact Hausdorff, so for him any compactification should be Hausdorff, and only Hausdorff spaces to start with can even hope to have a compactification in this sense.  Authors who only care about Hausdorff spaces will follow that without problem.

But I agree with @GeoffreySangston and the sources cited that if we don't assume compact means  compact Hausdorff, it makes sense to talk about compactification in general (i.e. compact spaces containing the original space as a dense subspace). And the Alexandroff one is a perfectly well defined construction.

**Comment by prabau** [comment]
@GeoffreySangston 
> One constraint is the desire to follow the structure of the description of [Line with two origins](https://topology.pi-base.org/spaces/S83). The issue is I'm conflating an equivalence class with a representative. To do it properly I think I would have to add text to the first paragraph, but when I tried to do that just now it seemed strictly worse, so I've deleted that attempt and added the current small change.

For the second paragraph, I still don't like the mod 2-pi business.  I think one way that would satisfactorily indicate what is meant, coming right after the first paragraph, would be:

... exactly when $\theta$ is not the "origin" point $0$.

(with quotes around origin, to make the reader pause for a second).  Possibly also add quotes around it in the first paragraph?

**Comment by GeoffreySangston** [comment]
I edited those in. If @Moniker1998 still rejects the "Alexandroff one-point compactification" idea, then we may need another opinion. @StevenClontz?

**Comment by StevenClontz** [comment]
Let's move discussion of the term "compactification" to #1174.

**Comment by prabau** [comment]
> I edited those in (please edit the second paragraph issue directly if it's still not up to snuff @prabau, if you don't mind).

@GeoffreySangston Where am I supposed to look?

**Comment by GeoffreySangston** [comment]
@prabau I was referring to my updates addressing your suggestions:

> For the second paragraph, I still don't like the mod 2-pi business. I think one way that would satisfactorily indicate what is meant, coming right after the first paragraph, would be:
> 
> ... exactly when θ is not the "origin" point 0 .
> 
> (with quotes around origin, to make the reader pause for a second). Possibly also add quotes around it in the first paragraph? We may be overthinking this.

**Comment by prabau** [comment]
> I was referring to my updates addressing your suggestions:

@GeoffreySangston Sure, but sorry I am lost.  Did you mean that you edited in place another comment in this conversation?  Link?  Or some other PR or issue?

**Comment by GeoffreySangston** [comment]
@prabau Oh sorry about that. I meant with the PR as it stands right now. Except the commit I thought I submitted must not have gone through (or more likely I got distracted right when I finished it). Let me try again.

**Comment by GeoffreySangston** [comment]
I'm just going to delete the branch and go with a new one to keep things simple.
